src/main/java/org/betonquest/betonquest/quest/event/run/RunEvent.java

```diff
@@ -0,0 +1,37 @@
+package org.betonquest.betonquest.quest.event.run;
+
+import org.betonquest.betonquest.api.QuestEvent;
+import org.betonquest.betonquest.api.profiles.Profile;
+import org.betonquest.betonquest.api.quest.event.nullable.NullableEvent;
+import org.betonquest.betonquest.exceptions.QuestRuntimeException;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Allows for running multiple events.
+ */
+public class RunEvent implements NullableEvent {
+
+    /**
+     * Events that the run event will execute.
+     */
+    private final List<QuestEvent> events = new ArrayList<>();
+
+    /**
+     * Create a run event from the given instruction.
+     *
+     * @param events events to run
+     */
+    public RunEvent(final List<QuestEvent> events) {
+        this.events.addAll(events);
```

Comment:
You could just List.copyOf()

Reply:
And why copy at all?

Reply:
For funny usage you could also not copy the list.